libcxx/test/std/utilities/expected/expected.expected/observers/has_value_clobber.pass.cpp

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Can you please put this test inside `has_value.pass.cpp`? You can just add another `{ ... }` block inside `test()` over there.

Reply:
done!